### ISSUE TITLE
2023-01-24_BugErrorMessage

### ISSUE THREAD

**Issue by Urpicha** [sviatoslav-lebediev/anki-quizlet-importer-extended#51]
{"statusCode":500,"code":"ERR_NON_2XX_3XX_RESPONSE","error":"Internal Server Error","message":"Response code 403 (Forbidden)"}

**Comment by sviatoslav-lebediev** [comment]
hi, could you please provide a quizlet deck ?

**Comment by med2580** [comment]
hi I've got the same message 
{"statusCode":500,"code":"ERR_NON_2XX_3XX_RESPONSE","error":"Internal Server Error","message":"Response code 403 (Forbidden)"}
https://quizlet.com/gb/622489288/session-1-gi-flash-cards/
that is the link for the deck

**Comment by sviatoslav-lebediev** [comment]
I've tried and looks like I can import this deck. If you get this error usually it means that "something wrong" with your network. "Something wrong" means that for some reason Quizlet doesn't work for you. I would try to use a VPN.
Here is a screenshot from the https://ankiweb.net/shared/info/1362209126 maybe it will help you

![image](https://user-images.githubusercontent.com/19693768/215471516-f8ca600a-7ebe-4d82-983b-a9959ee501e0.png)

List of items that I've imported

![image](https://user-images.githubusercontent.com/19693768/215470994-5f0b4f65-0ba8-4837-8864-6d75da02062d.png)
